docs/ref/checks.txt

```diff
@@ -912,14 +912,20 @@ The following checks are performed when a model contains a
 ------------
 
 The following checks are performed on :mod:`django.contrib.postgres` model
-fields:
+fields, indexes, and constraints:
 
 * **postgres.E001**: Base field for array has errors: ...
 * **postgres.E002**: Base field for array cannot be a related field.
 * **postgres.E003**: ``<field>`` default should be a callable instead of an
   instance so that it's not shared between all field instances. *This check was
   changed to* ``fields.E010`` *in Django 3.1*.
 * **postgres.W004**: Base field for array has warnings: ...
+* **postgres.E005**: 'django.contrib.postgres' must be in ``INSTALLED_APPS`` in
+  order to use ``<PostgreSQL-specific index>``.
+* **postgres.E006**: 'django.contrib.postgres' must be in ``INSTALLED_APPS`` in
+  order to use ``ExclusionConstraint``.
+* **postgres.E007**: 'django.contrib.postgres' must be in ``INSTALLED_APPS`` in
+  order to use ``<PostgreSQL-specific field>``.
```

Comment:
Any thoughts about using a single error code (e.g. `postgres.0005`) for all of them with a generic message?